File: python-programs/SpecialTwoDigitNumberLbp23.py
input = int(input())
original = input;
firstDigit = input % 10;
input = input // 10;
secondDigit = input % 10;
add = firstDigit + secondDigit;
product = firstDigit * secondDigit;
if original == (add + product):
    print("Yes")
else:
    print("No")

File: c-programs/SpecialTwoDigitNumberLbp23.c
#include <math.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    int original, sum, product, input, firstDigit, secondDigit;
    scanf("%d", &input);
    original = input;
    firstDigit = input % 10;
    input = input / 10;
    secondDigit = input % 10;
    sum = firstDigit + secondDigit;
    product = firstDigit * secondDigit;
    if (original == (sum + product)) {
        printf("%s", "Yes");
    } else {
        printf("%s", "No");
    }
    original = input;

    return 0;
}